site2/docs/client-libraries-java.md

```diff
@@ -1654,3 +1341,140 @@ PulsarClient client = PulsarClient.builder()
     .build();
 ```
 
+## Cluster-level failover
+
+More introduction for cluster-level failover please look at [Cluster-level failover](concepts-cluster-level-failover.md) chapter. 
```

Comment:
```suggestion
For more concepts and reference information about cluster-level failover, see [Cluster-level failover](concepts-cluster-level-failover.md). 
```

Reply:
@labuladong can you pls attach the screenshots for these updates in the PR description?